### PR TITLE
Additional UI/UX

### PR DESCRIPTION
**Pull request by katieli16**
Implemented the following queries:

- Get  all arriving flights for a given airport

- Get all departing flights for a given airport 

- Get total distance on all customer's itineraries

- Get number of airplanes maintained by each airport

### CODE REVIEW

File: templates/db.html [hunk 1]
```diff
@@ -31,7 +33,49 @@ <h2>Flights</h2>
...
+                    <td>{{ row['departtime'] }}</td>
+                    <td>{{ row['status'] }}</td>
+                </tr>
+                {% endfor %}
+            </table>
+            {% endif %}
+            
             <div class="filter">
                 <form action="/flightUI" method="POST">
                     <p> <label>Update</label>
```
Comment (maknoon) [inline templates/db.html]: this box should be removed below the two filter buttons

File: templates/db.html [hunk 2]
```diff
@@ -48,6 +92,18 @@ <h2>Flights</h2>
                     <input type="hidden" name="delayed" value=True />
                     <button>Get Delayed Flights</button>
                 </form>
+                <form action="/flightUI" method="GET">
```
Comment (maknoon) [inline templates/db.html]: this form should be a `POST`, not a `GET`

File: templates/db.html [hunk 3]
```diff
@@ -48,6 +92,18 @@ <h2>Flights</h2>
                     <input type="hidden" name="delayed" value=True />
                     <button>Get Delayed Flights</button>
                 </form>
+                <form action="/flightUI" method="GET">
+                <p> <label>Filter by Arriving Flights to Airport</label>
+                    <input type="text" name="ap_id" placeholder="Airport_ID" />
+                    <input type='submit' name="filterarriving" value="Submit" />
+                </p>
+                </form>
+                <form action="/flightUI" method="GET">
```
Comment (maknoon) [inline templates/db.html]: this form should be a `POST`, not a `GET`

**Review by maknoon — CHANGES_REQUESTED** [review]
Looks good

File: templates/db.html (hunk 1 above)
Comment (katieli16) [reply templates/db.html]: I'm not sure what you are you referring to

**Review by maknoon — APPROVED** [review]

**Pull request merged**